File: 01_arithmetic/01_real_numbers/01_real_numbers_properties/RealNumbersProperties.java
public class RealNumbersProperties {
    
    public static int closureAddition(int a, int b){
        return a + b;
    }

    public static int closureMultiplication(int a, int b){
        return a * b;
    }

    public static void main(String[] args) {
        System.out.println("Closure Addition: " + closureAddition(3, 5));
        System.out.println("Closure Multiplication: " + closureMultiplication(2, -3));
    }

}

File: 01_arithmetic/01_real_numbers/01_real_numbers_properties/real_numbers_properties.php
<?php

function closure_addition($a, $b){
    return $a + $b;
}

function closure_multiplication($a, $b){
    return $a * $b;
}

echo "Closure Addition: " . closure_addition(3, 5) . "\n";
echo "Closure Multiplication: " . closure_multiplication(2, -3);